igvc_ws/src/sensors/src/sensors_webhook.py

```diff
@@ -18,38 +19,71 @@
+
+POST:
+    The POST method is used to update the publish an IMU message to ROS and other
+    nodes.
+    To use submit the URL as is using POST, for the body inclue a JSON string that
+    is in the same format as the custom accelerometer message used for ROS. For 
+    more information about the structure of the message look at the custom message
+    reference page on the project wiki
+'''
 @app.route('/webhook/accelerometer', methods=['GET', 'POST'])
 def accel():
     global acceleration
     if request.method == 'POST':
         data = request.data
-        json = j.loads(data)
-        acceleration = json
-        rospy.loginfo('/webhook/accelerometer{' +  'type: ' + str(json['type']) + ', x: ' + str(json['x']) + ', y: ' + str(json['y']) + ', z: ' + str(json['z']) + '}')
-        return 'success'
+        json_data = json.loads(data)
+        acceleration = json_data
+        rospy.loginfo('/webhook/accelerometer{' +  'type: ' + str(json_data['type']) + ', x: ' + str(json_data['x']) + ', y: ' + str(json_data['y']) + ', z: ' + str(json_data['z']) + '}')
+        return jsonify({'status': 'success'})
     elif request.method == 'GET':
-        #rospy.loginfo("Getting gps location")
         return jsonify(acceleration)
 
 
+''' This is the GPS webhook, updates and gets the last known GPS location about the
+    To use this webhook you can submit either an GET or a POST method.
+
+GET:
+    If you want to submit a GET method, you can submit the URL as is. The GET method
+    will return the last known GPS location as a JSON string using the same format 
+    as we use in the custom message.
+
+POST:
+    Submit the URL as is with the body as a JSON string in the format of the custom 
+    GPS message used in ROS to update the GPS location. We will then publish the 
+    message to ROS on the topic 'gps_location'
+'''
 @app.route('/webhook/gps', methods=['GET', 'POST'])
 def gps():
     global gps_location
     if request.method == 'POST':
-        json = j.loads(request.data)
-        gps_location = json
-        rospy.loginfo('/webhook/gps{' + 'latitude: ' +  str(json['latitude']) + ', longitude: ' + str(json['longitude']) + ', altitude: ' + str(json['altitude']) + ', accuracy: ' + str(json['accuracy']) + ' speed: ' + str(json['speed']) + ', speed_accuracy: ' +  str(json['speed_accuracy']) + '}')
+        json_data = json.loads(request.data)
+        gps_location = json_data
+        rospy.loginfo('/webhook/gps{' + 'latitude: ' +  str(json_data['latitude']) + ', longitude: ' + str(json_data['longitude']) + ', altitude: ' + str(json_data['altitude']) + ', accuracy: ' + str(json_data['accuracy']) + ' speed: ' + str(json_data['speed']) + ', speed_accuracy: ' +  str(json_data['speed_accuracy']) + '}')
         
-        if publisher is not None:
-            publisher.send(ros.json_to_msg(request.data, msg.gps))
+    if publisher is not None:
+        publisher.send(ros.json_to_msg(request.data, msgs.gps))
 
         return 'success'
```

Comment:
I still wouldn't return 'success' or failed here as a string, either stringify it like you have for other functions or return a boolean. Usually for return a boolean is better.